src/driver/capacitor/CapacitorConnectionOptions.ts

```diff
@@ -9,15 +9,30 @@ export interface CapacitorConnectionOptions extends BaseConnectionOptions {
      */
     readonly type: "capacitor";
 
+    /**
+     * The capacitor-sqlite instance. For example, `new SQLiteConnection(CapacitorSQLite)`.
+     */
+    readonly driver: any;
+
     /**
      * Database name (capacitor-sqlite will add the suffix `SQLite.db`)
      */
     readonly database: string;
 
     /**
-     * The capacitor-sqlite instance. For example, `new SQLiteConnection(CapacitorSQLite)`.
+     * Set to true for database encryption
      */
-    readonly driver: any;
+    readonly encrypted?: boolean;
```

Comment:
Can't this just be `databaseMode !== undefined && databaseMode !== "no-encryption"`

Reply:
i get what you mean. this mirrors the options that are available for capacitor-sqlite. i thought as well that they are redundant. the thinking is that in the end its better to stay close to the underlying driver.

Reply:
I feel like that's just going to add extra confusion, isn't it?  

Setting `encrypted` to false just sets it to `no-encryption`.

https://github.com/capacitor-community/sqlite/blob/master/android/src/main/java/com/getcapacitor/community/database/sqlite/CapacitorSQLitePlugin.java#L168

I get that you're trying to mirror it but we're also trying to give a better API on top of capacitor, aren't we?

Reply:
good catch from the capacitor-sqlite source. it's ok for me to remove the `encrypted` property. i pushed an update.